java-manta-it/src/test/java/com/joyent/manta/client/MantaClientFindIT.java

```diff
@@ -32,7 +32,7 @@
 
 import static com.joyent.manta.client.MantaClient.SEPARATOR;
 
-@Test
+@Test(groups = { "nightly" })
```

Comment:
Are you sure we want the default setting for all new tests in this class to be "nightly"?